ldk/javascript/package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@oliveai/ldk",
-  "version": "3.7.0",
+  "version": "3.9.0",
```

Comment:
not sure about these changes, npm install did it

Reply:
I think this is fine. It looks like someone updated the `package.json` version without updating `package-lock.json`'s (or running `npm i`). So while it's not related, I think this is good to include